src/js/vic-v2/containers/ConfirmationPage.jsx

```diff
@@ -34,21 +37,39 @@ class ConfirmationPage extends React.Component {
     const response = form.submission.response
       ? form.submission.response.attributes
       : {};
+    const { veteranFullName, verified } = form.data;
+    const submittedAt = moment(form.submission.submittedAt);
 
     return (
       <div>
-        <h3 className="confirmation-page-title">Claim received</h3>
-        <p>We process claims in the order we receive them.</p>
-        <p>
-          We may contact you for more information or documents.<br/>
-          <i>Please print this page for your records.</i>
-        </p>
+        <p>You have completed your application.<br/>
+        We process applications and print cards in the order we receive them.</p>
+
+        <h4>What happens after I apply?</h4>
+        {!verified && <div>
+          <p>We’ll review your application to verify your eligibility. We may contact you if we have any questions or need more information.</p>
+          <p>You must have separated under honorable conditions to be eligible for a Veteran ID Card. If you have an other than honorable discharge, you can apply for an upgrade or correction. If your application goes through and your discharge is upgraded, you’ll be eligible for the Veteran ID Card. <a href="/discharge-upgrade-instructions" target="_blank">Find out how to apply for a discharge upgrade</a>.</p>
+          <p>If you uploaded a copy of your DD214, we can use it to review your application and verify that you were honorably discharged.</p>
+        </div>}
+        {verified && <div>
+          <p>You should receive your Veteran ID Card in the mail in about 60 days.<br/>
+          In the meantime, you can print and download a temporary digital Veteran ID Card.</p>
```

Comment:
We mention both print and download here, but only mention print below, should additional download guidance be provided as well? Or is that beyond the scope of this content?

Reply:
That does seem weird, @peggygannon you have any thoughts?